views/layouts/main.handlebars

```diff
@@ -29,16 +29,21 @@
     <a href="https://qsurvey.mozilla.com/s3/txp-firefox-send" rel="noreferrer noopener" class="feedback" target="_blank" data-l10n-id="siteFeedback">Feedback</a>
   </header>
   <div class="all">
+    <noscript>
+      <h2>Firefox Send requires JavaScript</h2>
+      <p><a href="https://github.com/mozilla/send/blob/master/docs/faq.md#why-does-firefox-send-require-javascript" target="_blank" rel="noreferrer noopener">Why does Firefox Send require JavaScript?</a></p>
+      <p>Please enable JavaScript and try again.</p>
+    </noscript>
     {{{body}}}
   </div>
   <div class="footer">
     <div class="legal-links">
       <a href="https://www.mozilla.org"><img class="mozilla-logo" src="/resources/mozilla-logo.svg"/></a>
-      <a href="https://www.mozilla.org/about/legal" data-l10n-id="footerLinkLegal"></a>
-      <a href="https://testpilot.firefox.com/about" data-l10n-id="footerLinkAbout"></a>
-      <a href="/legal" data-l10n-id="footerLinkPrivacy"></a>
-      <a href="/legal" data-l10n-id="footerLinkTerms"></a>
-      <a href="https://www.mozilla.org/privacy/websites/#cookies" data-l10n-id="footerLinkCookies"></a>
+      <a href="https://www.mozilla.org/about/legal" data-l10n-id="footerLinkLegal">Legal</a>
+      <a href="https://testpilot.firefox.com/about" data-l10n-id="footerLinkAbout">About Test Pilot</a>
+      <a href="/legal" data-l10n-id="footerLinkPrivacy">Privacy</a>
+      <a href="/legal" data-l10n-id="footerLinkTerms">Terms</a>
+      <a href="https://www.mozilla.org/privacy/websites/#cookies" data-l10n-id="footerLinkCookies">Cookies</a>
```

Comment:
Again, all the defaults for the footer link text are in en-US, but will still localize correctly w/ l20n if they have JavaScript enabled.